components/Popups.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
```

Comment:
are we even using this file? if not, remove

Reply:
Was Wendy s file